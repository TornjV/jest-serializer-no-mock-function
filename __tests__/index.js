test('Mock test', () => {
  const mock = jest.fn().mockName('myMock');

  mock('hello', { foo: 'bar' });

  expect(mock).toMatchSnapshot();
});