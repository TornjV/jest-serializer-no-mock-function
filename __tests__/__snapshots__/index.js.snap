// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mock test 1`] = `[Function]`;
